libs/sr/ui/src/Optimization/OptimizationTargetSelector.tsx

```diff
@@ -13,22 +14,24 @@ export function OptimizationTargetSelector({
   setOptTarget: (o: Read) => void
 }) {
   const { t } = useTranslation('optimize')
-  const { calc } = useCalcContext()
+  const calc = useContext(CalcContext)
```

Comment:
Can we create a `useSrCalcContext` hook that calls `useContext(CalcContext)` and casts it to srcalc for us?